### ISSUE TITLE
No module named 'rotated_roi'

### ISSUE THREAD

**Issue by adnantariq18** [weijiawu/TransDETR#10]
Hi there I am facing the issue in this module as i have already installed the dependency which is as follow

cd ./models/Rotated_ROIAlign
python setup.py build_ext --inplace

dependent lib ninja was also installed as my main goal was to get the model and infer the pre-train model

**Comment by adnantariq18** [comment]
![Screenshot from 2022-05-31 09-49-32](https://user-images.githubusercontent.com/102550268/171095107-9b0cd92b-9131-4bb1-84ff-d217f6f682b6.png)

**Comment by weijiawu** [comment]
please refer to the [issue](https://github.com/weijiawu/TransDETR/issues/8)

**Comment by adnantariq18** [comment]
i did tried that method but i think i was not able to get the exact path could you please provide the specific path i used the from python 
import os
os.environ['PYTHONPATH']+=":/content/TransDETR/models/Rotated_ROIAlign/build/lib.linux-x86_64-3.7"
and didn't work well
and also i tried form terminal
 export PYTHONPATH=$PYTHONPATH: ./Rotated_ROIAlign/build/lib.linux-x86_64-3.7
-bash: export: `./Rotated_ROIAlign/build/lib.linux-x86_64-3.7': not a valid identifier

**Comment by weijiawu** [comment]
> i did tried that method but i think i was not able to get the exact path could you please provide the specific path i used the from python import os os.environ['PYTHONPATH']+=":/content/TransDETR/models/Rotated_ROIAlign/build/lib.linux-x86_64-3.7" and didn't work well and also i tried form terminal input command : export PYTHONPATH=$PYTHONPATH: ./Rotated_ROIAlign/build/lib.linux-x86_64-3.7 error on terminal : -bash: export: `./Rotated_ROIAlign/build/lib.linux-x86_64-3.7': not a valid identifier

Anyway, you need to add the path to the python default path. 

https://www.edureka.co/blog/add-python-to-path/ maybe useful for you.

**Comment by adnantariq18** [comment]
yes i was able to add the python path in it and got the output of rotated_roi  the screenshot of the output after installing .. you can in the picture .. is it the correct output?

**Comment by adnantariq18** [comment]
i think i will require to install rotated_roi properly .. are there any dependencies and versions of the it

**Comment by CccJ23** [comment]
> 我用这个https://github.com/looput/Rotated_ROIAlign.git来安装 rotated_roi 并在这个存储库中替换

请问您解决了问题吗

**Comment by P1ng3609** [comment]
> > 我用这个[https://github.com/looput/Rotated_ROIAlign.git来安装](https://github.com/looput/Rotated_ROIAlign.git%E6%9D%A5%E5%AE%89%E8%A3%85) rotated_roi 并在这个存储库中替换
> 
> 请问您解决了问题吗

请问您解决了这个问题了嘛？

**Comment by P1ng3609** [comment]
> > 我用这个[https://github.com/looput/Rotated_ROIAlign.git来安装](https://github.com/looput/Rotated_ROIAlign.git%E6%9D%A5%E5%AE%89%E8%A3%85) rotated_roi 并在这个存储库中替换
> 
> 请问您解决了问题吗

请问您解决了这个问题了嘛？
